stack.yaml

```diff
@@ -4,20 +4,21 @@ packages:
   - .
 
 extra-deps:
-  - ghc-lib-parser-9.2.3.20220507
-  - ghc-lib-parser-ex-9.2.0.4
+  - ghc-lib-parser-9.2.3.20220527
+  - ghc-lib-parser-ex-9.2.1.0
```

Comment:
changing this though is not, it goes together with the change in default of `ghc-lib`.